store/postgres/src/relational_queries.rs

```diff
@@ -2560,7 +2595,9 @@ impl<'a> FilterQuery<'a> {
         write_column_names(&column_names, &table, &mut out)?;
         self.filtered_rows(table, filter, out.reborrow())?;
         out.push_sql("\n ");
-        self.sort_key.order_by(&mut out, Some("c."))?;
+        let join_table_collection = filter.as_ref().map(|f| &f.table_join_collection);
```

Comment:
- [ ] allow to extend the JoinTableCollection with SortKey (prefix won't be available if EntityFilter::Child is not used)